index.html

```diff
@@ -176,6 +176,24 @@ <h2 class="about-heading-small">Skills</h2>
         </li>
         <hr>
     </ul>
+    <!-- Contact Me Section Starts -->
+    <div id="contact" class="contact-me-form-container">
+        <h1 class="contact-me-form-heading">Contact me</h1>
+        <p class="contact-me-form-paragraph">If you have an application you are interested in developing, a
+            feature that you need built
+            or a project that
+            needs coding. I'd love to help with it</p>
+        <form action="https://formspree.io/f/mqkoaqjr" method="POST" class="contact-me-form">
+            <input class="contact-me-form-text-input" required maxlength="30" name="name" type="text"
```

Comment:
<img width="884" alt="1" src="https://user-images.githubusercontent.com/77843050/221130329-3b81eb20-a66e-4201-bfc9-14b20022a5f6.png">

- [ ] You have done what we `EXPECTED` to see, which was lovely and genius from you ❤️, but the minor issue is in the design above, please make sure you adjust the padding on the left and add the black line at the bottom of the page and request for a new review